spark/src/test/java/org/apache/iceberg/spark/source/TestSparkDataWrite.java

```diff
@@ -374,6 +375,58 @@ public void testPartitionedCreateWithTargetFileSizeViaOption() throws IOExceptio
     }
   }
 
+  @Test
+  public void testPartitionedFanoutCreateWithTargetFileSizeViaOption() throws IOException {
```

Comment:
In general,  I think we could abstract a common method between this method and `testPartitionedCreateWithTargetFileSizeViaOption`,  they are almost the same.

Reply:
Well, let me change it.